src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: () => {
+    },
+    removeBook: () => {
+    },
```

Comment:
- [ ] Please complete these reducers for adding and removing a book from the state as the requirements asked.

Reply:
Thank you for reviewing my PR👍🏽. Cheer and happy coding😎